apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.mustache

```diff
@@ -17,28 +17,56 @@
+                    class="form-control big-o-input-input"
+                    size="{{size}}"
+                    {{^editable}}disabled{{/editable}}
+                    {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}}
+                    placeholder="{{placeholder}}"
+                    aria-describedby="pl-big-o-input-{{uuid}}-label pl-integer-input-{{uuid}}-suffix"
+                />
                 <span class="input-group-append">
-                    <span class="input-group-text">$)$</span>
+                    <span
+                        class="input-group-text"
+                        id="pl-big-o-input-{{uuid}}-suffix">$)$</span>
                     {{#show_info}}
                     <a role="button" class="btn btn-light border d-flex align-items-center" data-toggle="popover" data-html="true" title="Symbolic" data-content="{{info}}" data-placement="auto" data-trigger="focus" tabindex="0">
-                    <i class="fa fa-question-circle" aria-hidden="true"></i>
-                    {{/show_info}}
-                    {{^show_info}}
-                    <span style="display: flex; align-items: center;">
-                    {{/show_info}}
-
-                    {{#correct}}&nbsp;<span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
-                    {{#partial}}&nbsp;<span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i> {{partial}}%</span>{{/partial}}
-                    {{#incorrect}}&nbsp;<span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>{{/incorrect}}
-                    {{#show_info}}
+                        <i class="fa fa-question-circle" aria-hidden="true"></i>
                     </a>
                     {{/show_info}}
-                    {{^show_info}}
-                    </span>
-                    {{/show_info}}
+                    {{#correct}}
+                        <span class="input-group-text">
+                            <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>
+                        </span>
+                    {{/correct}}
+                    {{#partial}}
+                        <span class="input-group-text">
+                            <span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i> {{partial}}%</span>
+                        </span>
+                    {{/partial}}
+                    {{#incorrect}}
+                        <span class="input-group-text">
+                            <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>
+                        </span>
+                    {{/incorrect}}
+                    {{#parse_error}}
+                        <a role="button"
+                        class="btn btn-light border d-flex align-items-center text-danger"
```

Comment:
Micro-nit: would be good to keep consistent formatting when breaking up tags into multiple lines.

```suggestion
                        <a
                            role="button"
                            class="btn btn-light border d-flex align-items-center text-danger"
```

...and so on.

Reply:
Times like this I wish there was nice autoformatting for mustache

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/8191/commits/c745fc6519998c435e1a8d371804abfa3691789a